drivers/isl29501/isl29501.cpp

```diff
@@ -339,8 +339,8 @@ void ISL29501::_measure(void)
 	distance = ((data_msb << 8) + data_lsb) / 2000.0f;
 	//correct distance from errors due to phase-distance inconsistency
 	float phase = (distance / 33.3f) * M_PI_F * 2.0f;
-	distance += (1.1637 * pow(phase, 3) - 3.8654 * pow(phase, 2) + 1.3796 * phase - 0.0436);
-	DF_LOG_ERR("distance: %f offset: 0x%x ", distance, offset);
+	distance += (1.1637f * (float)pow(phase, 3) - 3.8654f * (float)pow(phase, 2) + 1.3796f * phase - 0.0436f);
```

Comment:
Don't use pow at all. use phase * phase * phase and phase * phase. Pow is for cases like phase ^ 0.5, not for natural numbers. This should not have been merged without addressing my comment.